### PR TITLE
1696 fix ios x button

### PR DESCRIPTION
**Pull request by edmofro**

### CODE REVIEW

File: packages/meditrak-app/app/navigation/Navigator.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ import {
   ROUTES_WITH_INVISIBLE_HEADERS,
 } from './constants';
 import { SyncContainer } from '../sync';
-import { THEME_COLOR_ONE, THEME_COLOR_THREE } from '../globalStyles';
+import { THEME_COLOR_ONE } from '../globalStyles';
```
Comment (edmofro) [inline packages/meditrak-app/app/navigation/Navigator.js:35]: Was unused

File: packages/meditrak-app/app/sync/SyncPage.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import React from 'react';
 import { View, StyleSheet } from 'react-native';
 import PropTypes from 'prop-types';
 import { TupaiaBackground, Button, Text, ProgressBar } from '../widgets';
-import { formatPlural, formatDate } from '../utilities';
+import { formatDate } from '../utilities';
```
Comment (edmofro) [inline packages/meditrak-app/app/sync/SyncPage.js:10]: Was unused

File: packages/meditrak-app/app/assessment/DumbSurveyScreen.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ const LENGTH_OF_TRANSITION = 300;
 export class DumbSurveyScreen extends React.Component {
   static navigationOptions = ({ navigation }) => ({
     headerTitle: navigation.state.params.headerLabel,
-    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')} />,
+    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')} labelVisible={false} />,
```
Comment (chris-bes) [inline packages/meditrak-app/app/assessment/DumbSurveyScreen.js:31]: I'm curious why we didn't need to put this for the default back buttons (ones that we don't have to replace the back image)...

**Review by chris-bes — APPROVED** [review]
LGTM!

File: packages/meditrak-app/app/assessment/DumbSurveyScreen.js (hunk 3 above)
Comment (edmofro) [reply packages/meditrak-app/app/assessment/DumbSurveyScreen.js:31]: Also curious, but don't know!

**Pull request merged**